### DISCUSSION TITLE
Can't see all the hrr results in each time step

### DISCUSSION THREAD

**Discussion by HunterXHunter1997** [firemodels/fds#13148] [Q&A]
Hello team,

I am doing a heptane pool fire, but I can't see all the hrr values in each time step of my simulation.
I did two simulations: one with 100sec, which looks good.
And another one with 1800sec, which I have a problem in.
So should I change the time step? or the mesh on the 1800sec?
Thanks.

**Comment by drjfloyd** [comment]
Is your issue that you wish to have more lines of output in the HRR csv file? In which case see Chapter 22 of the User's Guide

or is your issue something else?

**Comment by HunterXHunter1997** [comment]
Thanks for the quick reply,
yes it is a part of my problem, I will rerun my case and check if it is solved.
Thanks again
